app/src/main/java/org/fossasia/badgemagic/util/StorageUtils.kt

```diff
@@ -19,12 +19,12 @@ import org.fossasia.badgemagic.data.fragments.CONF_SPEED
 import org.fossasia.badgemagic.data.fragments.ConfigInfo
 import org.json.JSONObject
 
-object StorageUtils {
+class StorageUtils(val context: Context) {
     private val EXTERNAL_STORAGE_DIRECTORY = "${Environment.getExternalStorageDirectory()
```

Comment:
try to change Environment.getExternalStorageDirectory to app specific directory code now